addons/web/static/tests/views/calendar_tests.js

```diff
@@ -2604,6 +2606,39 @@ QUnit.module('Views', {
         calendar.destroy();
     });
 
+    QUnit.test('calendar sidebar toggle', function (assert) {
+        assert.expect(4);
+
+        var calendar = createView({
+            View: CalendarView,
+            model: 'event',
+            data: this.data,
+            arch:
+            '<calendar date_start="start_date">'+
+                    '<field name="name"/>'+
+            '</calendar>',
+            services: {
+                local_storage: AbstractStorageService.extend({storage: new RamStorage()}),
+            },
+        });
+
+        var fullWidth = calendar.call('local_storage', 'getItem', 'calendar_fullWidth');
+        assert.strictEqual(fullWidth, undefined,
+            "calendar_fullWidth should be undefined in local storage");
+        assert.containsNone(calendar, '.o_calendar_sidebar_container.o_sidebar_hidden',
+            "sidebar should be opened");
+
+        // click on close
+        testUtils.dom.click(calendar.$('.o_calendar_sidebar_toggler'));
+        fullWidth = calendar.call('local_storage', 'getItem', 'calendar_fullWidth');
+        assert.strictEqual(fullWidth, true,
+            "calendar_fullWidth should be true in local storage");
+        assert.containsOnce(calendar, '.o_calendar_sidebar_container.o_sidebar_hidden',
+            "sidebar should be hidden");
+
```

Comment:
maybe trigger a reload and check that the sidebar is still folded

Reply:
Done 👍